app/src/main/java/uk/ac/ebi/atlas/experimentimport/admin/SingleCellExperimentAdminController.java

```diff
@@ -17,4 +20,11 @@ public SingleCellExperimentAdminController(DataFileHub dataFileHub,
                         new ExperimentOpLogWriter(dataFileHub),
                         new SingleCellOpsExecutionService(scxaExperimentCrud)));
     }
+
+    @Caching(evict = {
+            @CacheEvict(cacheNames = "privateExperimentAccessions", allEntries = true)})
+    @GetMapping(value = "/json/experiments/private/experimentAccessions/clearCache")
```

Comment:
It's a minor change but i believe we can simplify the endpoint here. As it is just to evict private experiment cache and not returning anything we can rename it as `/private/experiments/cache/clear`